source/algorithms.tex

```diff
@@ -5421,7 +5422,7 @@
 template<class ForwardIterator1, class ForwardIterator2>
   constexpr ForwardIterator2
     swap_ranges(ForwardIterator1 first1, ForwardIterator1 last1,
-                ForwardIterator2 first2);                               // freestanding
```

Comment:
This was the wrong place for the freestanding marker before.  So this fixes the prior editorial mistake.

Reply:
Thanks!